engine/src/test/groovy/graphql/nadel/e2e/NadelE2ETest_SharedNamespaceWIP.groovy

```diff
@@ -0,0 +1,409 @@
+package graphql.nadel.e2e
+
+import graphql.nadel.Nadel
+import graphql.nadel.NadelExecutionInput
+import graphql.nadel.ServiceExecution
+import graphql.nadel.ServiceExecutionFactory
+import graphql.nadel.ServiceExecutionResult
+import graphql.nadel.engine.testutils.TestUtil
+import spock.lang.Specification
+
+import static graphql.nadel.NadelEngine.newNadel
+import static graphql.nadel.NadelExecutionInput.newNadelExecutionInput
+import static graphql.nadel.engine.testutils.TestUtil.typeDefinitions
+import static java.util.concurrent.CompletableFuture.completedFuture
+
+class NadelE2ETest_SharedNamespaceWIP extends Specification {
+
+    def "query with two services sharing a namespaced field"() {
+        def nsdl = [
+                Issues     : '''
+            service Issues {
+              
+              directive @namespaced on FIELD_DEFINITION
```

Comment:
Would moving the directive declaration to a common piece of DSL (not specific to a service) be more realistic?
It might not make any difference now, but perhaps it's more future proof? Like, if the way we merge service DSLs together changes or something like that.